attribution_scopes.md

```diff
@@ -89,9 +89,9 @@ If there are multiple sources whose `attribution_scopes` contains at least one o
 
 If the trigger registration's `attribution_scopes` is empty, then all sources are considered for attribution.
 
-Once an `attribution_scope_limit` is set, the last K values (where K = `attribution_scope_limit`) of `attribution_scopes` will be considered the final set of `attribution_scopes` values and any source with additional `attribution_scopes` values will be treated as if the attribution scopes were empty.
+Once an `attribution_scope_limit` is set, the last K values (where K = `attribution_scope_limit`) of `attribution_scopes` will be considered the final set of `attribution_scopes` values per reporting origin per destination site, and any source with additional `attribution_scopes` values will be treated as if the attribution scopes were empty.
```

Comment:
Per recent discussion, sources with those scopes will be deleted instead. Also shall we mention the following recent change?

1. If the new source uses the attribution scopes feature, all previous pending source registrations that do not use the feature will be deleted.
2. If the new source doesn't use the attribution scopes feature, all previous pending source registrations will be treated as if the feature is not used.

Reply:
I have updated the text to reflect this behavior @linnan-github